### ISSUE TITLE
Removing unnecessary lines from .travis.yml and fixing CMakeLists.txt

### ISSUE THREAD

**Issue by czgdp1807** [codezonediitj/BNN#25]
#### Description of the problem

1. The following line is not needed, https://github.com/codezonediitj/BNN/blob/d21065c1a7f1fac84c08819ef2aa866a63ef82ed/.travis.yml#L13 

2. The following should use 1.10.0 release, https://github.com/codezonediitj/BNN/blob/d21065c1a7f1fac84c08819ef2aa866a63ef82ed/CMakeLists.txt#L23-L31

**Comment by yaswanth2802** [comment]
can 1 work on this issue @czgdp1807

**Comment by czgdp1807** [comment]
Sure.

**Comment by yaswanth2802** [comment]
@czgdp1807 do we need to remove the lines from 23 to 31 in CMakeLists.txt and change the version to 1.10.0 if this is right can you tell me where we can change the version because I am unable to find the src file for changing the version

**Comment by czgdp1807** [comment]
No. We have to keep 23 to 31 in `CMakeLists.txt`. As of now Google test's master branch is used (see the URL). We need to change it to use `1.10.0`. The line to be removed `13` from `.travis.yml`.

**Comment by yaswanth2802** [comment]
I have changed the .travis.yml, and I have another doubt currently I am participating as a student in Kwoc 2022 so if I am changing the or contributing it will be counted right because there are no labels like Kwoc2022 like that.

**Comment by czgdp1807** [comment]
> I have another doubt currently I am participating as a student in Kwoc 2022 so if I am changing the or contributing it will be counted right because there are no labels like Kwoc2022 like that.

AFAIK, contributions should be counted towards KWoC automatically. Though please ask the organisers for a clearer answer.

**Comment by yaswanth2802** [comment]
> No. We have to keep 23 to 31 in `CMakeLists.txt`. As of now Google test's master branch is used (see the URL). We need to change it to use `1.10.0`. The line to be removed `13` from `.travis.yml`.

as of now, the available version is 1.11.0 but we need to change it to 1.10.0. if this is right can you tell me if we can use this command https://github.com/google/googletest/archive/release-1.10.0.tar.gz  then it will work or not in place of      https://github.com/google/googletest

**Comment by czgdp1807** [comment]
Try changing the GIT_TAG to `v1.10.x` from `master`. See if it works on your system.

**Comment by yaswanth2802** [comment]
sir how to cheak the working of this in our system

**Comment by czgdp1807** [comment]
Try the following steps (assuming you are currently in `BNN` directory),

1. `cd ../BNN`
2. `mkdir build-bnn`
3. `cd build-bnn`
4. `cmake -DBUILD_TESTS=ON -DINSTALL_GOOGLETEST=ON ../BNN`
5. `make`
6. `./bin/test_core` (it should pass)

Install `cmake`, `g++` before executing the above commands.

**Comment by yaswanth2802** [comment]
i am getting error while performing the 4th step sir

**Comment by czgdp1807** [comment]
Please copy and post the error here.
